packages/orbit-components/playroom.config.js

```diff
@@ -0,0 +1,88 @@
+const path = require("path");
+const { defaultFoundation } = require("@kiwicom/orbit-design-tokens");
+
+module.exports = {
+  title: "Orbit Playroom",
+  components: require.resolve("./playroomComponents.tsx"),
+  outputPath: "./.playroom/playroom",
+  frameComponent: "./playroom/FrameComponent.tsx",
+  snippets: "./playroom/snippets/index.ts",
+  scope: "./playroom/scope.ts",
+  port: 9000,
+  openBrowser: false,
+  widths: Object.values(defaultFoundation.breakpoint),
+  defaultVisibleWidths: [
+    defaultFoundation.breakpoint.mediumMobile,
+    defaultFoundation.breakpoint.desktop,
+  ],
+  baseUrl: "/playroom/",
+  webpackConfig: () => ({
+    resolve: {
+      extensions: [".js", ".jsx", ".ts", ".tsx", ".css"],
```

Comment:
Do we need .js* extensions?

Reply:
I believe playroom is actually checking for the built files, inside `lib`, so it's `js`. But I didn't check it back then. I would suggest we come back to this soon with this and other improvements